src/twisted/test/test_nooldstyle.py

```diff
@@ -24,6 +24,11 @@
     _skip = "Not running with TWISTED_NEWSTYLE=1"
 
 
+forbiddenModules = [
+    "twisted.internet",
```

Comment:
On second thought, maybe it *should* be called `forbiddenModules`, since it would be best to do this as narrowly as possible for its first iteration.